### PR TITLE
Use auto tag

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Use the @pxblue/tag script

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,40 +26,72 @@ jobs:
...
+            command:   |
+              cd routing-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing-typescript
+        - run:
+            name: Tag @pxblue/cra-template-authentication
+            command:  |
+              cd authentication && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication
+        - run:
+            name: Tag @pxblue/cra-template-routing-authentication-typescript
+            command:   |
+              cd authentication-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-routing-authentication-typescript
```
Comment (joebochill) [inline .circleci/config.yml:94]:
```suggestion
        - run:
            name: Tag @pxblue/cra-template-authentication-typescript
            command:   |
              cd authentication-typescript && yarn tag:package -b $CIRCLE_BRANCH -s -cra-template-authentication-typescript
```

Comment (joebochill) [reply .circleci/config.yml:94]: or you can pull out the suffixes while you're in here.

**Review by joebochill — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (emclaug2) [reply .circleci/config.yml]: I've removed the suffixes.

File: .circleci/config.yml [hunk 2]
```diff
@@ -24,40 +26,72 @@ jobs:
...
+        - run:
+            name: Tag @pxblue/cra-template-routing-typescript
+            command:   |
+              cd routing-typescript && yarn tag:package -b $CIRCLE_BRANCH
+        - run:
+            name: Tag @pxblue/cra-template-authentication
+            command:  |
+              cd authentication && yarn tag:package -b $CIRCLE_BRANCH
+        - run:
+            name: Tag @pxblue/cra-template-routing-authentication-typescript
```
Comment (joebochill) [inline .circleci/config.yml:92]:
```suggestion
            name: Tag @pxblue/cra-template-authentication-typescript
```

**Review by joebochill — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 3]
```diff
@@ -24,40 +26,72 @@ jobs:
...
+        - run:
+            name: Publish @pxblue/cra-template-routing-typescript
+            command:   |
+              cd routing-typescript && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-authentication
+            command:  |
+              cd authentication && yarn publish:package -b $CIRCLE_BRANCH
+        - run:
+            name: Publish @pxblue/cra-template-routing-authentication-typescript
```
Comment (joebochill) [inline .circleci/config.yml:61]:
```suggestion
            name: Publish @pxblue/cra-template-authentication-typescript
```

**Review by joebochill — APPROVED** [review]

**Pull request merged**